frb/halos.py

```diff
@@ -882,6 +882,40 @@ def rho_b(self, xyz):
         #
         return rho
 
+class MB15(ModifiedNFW):
+    """
+    Encodes the Galactic halo profile from
+    Miller & Bregman 2015, ApJ, 800, 14
+    https://ui.adsabs.harvard.edu/abs/2015ApJ...800...14M/abstract
+
+
+    """
+    def __init__(self, log_Mhalo=12.18, c=7.67, f_hot=0.75, **kwargs):
+        # Init ModifiedNFW
+        ModifiedNFW.__init__(self, log_Mhalo=log_Mhalo, c=c, f_hot=f_hot, **kwargs)
+
+        # Best parameters
+        self.beta = 0.45
+        self.n0_rc3b = 0.79e-2  # Last entry of Table 2; Crazy units
```

Comment:
Why choosing these two best fits of OVII? From their Table 2, second line, it seems like if use OVIII parameters, then n0_rc3b would be roughly twice as much? (p.s., I didn't read their paper throughout to understand their model... maybe you can add a line of comment here about the choice of parameters? )

Reply:
Actually, it looks like the mass doesn't vary so much in their 
range of models.  Adding a note.